test/canary-environment/tests/generic/makes_progress.sql

```diff
@@ -11,9 +11,11 @@
 -- {{ model }} must be mentioned somewhere in the query or dbt will croak */
 
 {% set query %}
+   SET CLUSTER = qa_canary_environment_compute;
    BEGIN;
    DECLARE c1 CURSOR FOR SUBSCRIBE ( SELECT * FROM {{ model }} )  WITH (SNAPSHOT = FALSE);
-   FETCH 1 c1 WITH (timeout='60s');
+   -- SUBSCRIBE takes extra-long time due to #22061
```

Comment:
I'd change that to a format we can recognize, like "TODO: Switch timeout back to 60s after #22061 is fixed"

Reply:
absolutely